content/compose/environment-variables/set-environment-variables.md

```diff
@@ -65,7 +65,7 @@ services:
       webapp:
         image: my-webapp-image
         environment:
-          - DEBUG=${DEBUG}
+          DEBUG=${DEBUG}
```

Comment:
This doesn't look valid. There's no colon delimiter for the map key/value, so this would be interpreted as a string.